pulsar-broker/src/test/java/org/apache/pulsar/broker/service/persistent/TopicDuplicationTest.java

```diff
@@ -529,6 +538,89 @@ public void testDisableNamespacePolicyTakeSnapshotShouldNotThrowException() thro
         persistentTopic.checkDeduplicationSnapshot();
     }
 
+    @Test
+    public void testFinishTakeSnapshotWhenTopicLoading() throws Exception {
```

Comment:
This test passed locally even without changes of `MessageDeduplication`, could you explain it?

<img width="744" alt="image" src="https://github.com/apache/pulsar/assets/18204803/ffe988f0-575a-4b2f-aa52-d961bd78ad8e">

Reply:
Test has been improved. Currently would not passed